server/service/src/requisition/common.rs

```diff
@@ -121,6 +122,7 @@ pub fn check_exceeded_max_orders_for_period(
                 .program_id(EqualFilter::equal_to(input.program_id))
                 .order_type(EqualFilter::equal_to(&order_type.name))
                 .period_id(EqualFilter::equal_to(input.period_id))
+                .store_id(EqualFilter::equal_to(&input.store_id))
```

Comment:
Side quest: Bug fix. Wasn't filtering by store so was checking all if you have multiple stores on site